src/modules/dashboard/components/ColonyHome/ColonyDomainSelector/CreateDomainButton.css

```diff
@@ -7,7 +7,7 @@
 }
 
 .main {
-  composes: button from "~styles/reset.css";
+  composes: button from '~styles/reset.css';
```

Comment:
I just checked and seems like there is no stylelint config set for string-quotes (https://stylelint.io/user-guide/rules/list/string-quotes/) so maybe we should set something up first before making these changes.

Reply:
Good spot, this was due to my prettier config. I've now disabled the single quotes for css.